### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/spark-on-k8s-operator/commit/e647fad39f81bdcf6d78584eb0382f554ac74866#commitcomment-49714646

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/spark-on-k8s-operator/commit/fd4491d57de99355c86d8784a28180965e8c6294

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -27,7 +27,7 @@ install-sparkctl: | sparkctl/sparkctl-darwin-amd64 sparkctl/sparkctl-linux-amd64
 		sudo cp sparkctl/sparkctl-linux-amd64 /usr/local/bin/sparkctl; \
 		sudo chmod +x /usr/local/bin/sparkctl; \
 	elif [ "$(UNAME)" = "darwin" ]; then \
-		echo "installing macos binary to /usr/local/bin/sparkctl"; \
+		echo "installing macOS binary to /usr/local/bin/sparkctl"; \
```
Comment (jsoref) [inline Makefile:30]: brand

File: docs/gcp.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ The connectors require certain Hadoop properties to be set properly to function.
 both through a custom Hadoop configuration file, namely, `core-site.xml` in a custom image, or via the `spec.hadoopConf` 
 section in a `SparkApplication`. The example Dockerfile mentioned above shows the use of a custom `core-site.xml` and a 
 custom `spark-env.sh` that points the environment variable `HADOOP_CONF_DIR` to the directory in the container where 
-`core-site.xml` is located. The example `core-sitem.xml` and `spark-env.sh` can be found 
+`core-site.xml` is located. The example `core-site.xml` and `spark-env.sh` can be found
```
Comment (jsoref) [inline docs/gcp.md:17]: file...

File: docs/gcp.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ The connectors require certain Hadoop properties to be set properly to function.
 both through a custom Hadoop configuration file, namely, `core-site.xml` in a custom image, or via the `spec.hadoopConf` 
 section in a `SparkApplication`. The example Dockerfile mentioned above shows the use of a custom `core-site.xml` and a 
 custom `spark-env.sh` that points the environment variable `HADOOP_CONF_DIR` to the directory in the container where 
-`core-site.xml` is located. The example `core-sitem.xml` and `spark-env.sh` can be found 
+`core-site.xml` is located. The example `core-site.xml` and `spark-env.sh` can be found 
 [here](https://github.com/GoogleCloudPlatform/spark-on-k8s-gcp-examples/tree/master/conf).
```
Comment (jsoref) [inline docs/gcp.md:18]: Note that `here` is not good style. https://www.w3.org/TR/WCAG20-TECHS/F84.html

I may at some point send a docs PR (it's really outside the scope of this PR, the whitespace change at the end of this is really out of scope too)

File: docs/user-guide.md [hunk 4]
```diff
@@ -447,7 +447,7 @@ Note that the mutating admission webhook is needed to use this feature. Please r
 ### Using Security Context
 
 A `SparkApplication` can specify a `SecurityContext` for the driver or executor containers, using the optional field `.spec.driver.securityContext` or `.spec.executor.securityContext`.
-`SparkApplication` can also specify a `PodSecurityContext` for the driver or executor pod, using the optional field `.spec.driver.PodsecurityContext` or `.spec.executor.PodsecurityContext`. Below is an example:
+`SparkApplication` can also specify a `PodSecurityContext` for the driver or executor pod, using the optional field `.spec.driver.podSecurityContext` or `.spec.executor.podSecurityContext`. Below is an example:
```
Comment (jsoref) [inline docs/user-guide.md:450]: these reference an item 5 lines below

File: docs/who-is-using.md [hunk 5]
```diff
@@ -1,18 +1,18 @@
 ## Who Is Using the Kubernetes Operator for Apache Spark?
 
-| Organization | Contact (Github User Name) | Environment | Description of Use |
+| Organization | Contact (GitHub User Name) | Environment | Description of Use |
```
Comment (jsoref) [inline docs/who-is-using.md:3]: brand

File: docs/who-is-using.md [hunk 6]
```diff
@@ -1,18 +1,18 @@
...
 | ------------- | ------------- | ------------- | ------------- |
 | [Caicloud](https://intl.caicloud.io/) | @gaocegege | Production | Cloud-Native AI Platform |
-| Microsoft (MileIQ) |@dharmeshkakadia| Production | AI & Analytics |
-| Lightbend |@yuchaoran2011| Production | Data Infrastructure & Operations |
+| Microsoft (MileIQ) | @dharmeshkakadia | Production | AI & Analytics |
```
Comment (jsoref) [inline docs/who-is-using.md:6]: The rest of these are whitespace. The file will not be checked by my tool going forward.

File: pkg/controller/sparkapplication/controller.go [hunk 7]
```diff
@@ -177,7 +177,7 @@ func (c *Controller) Stop() {
 // Callback function called when a new SparkApplication object gets created.
 func (c *Controller) onAdd(obj interface{}) {
 	app := obj.(*v1beta2.SparkApplication)
-	glog.Infof("SparkApplication %s/%s was added, enqueueing it for submission", app.Namespace, app.Name)
+	glog.Infof("SparkApplication %s/%s was added, enqueuing it for submission", app.Namespace, app.Name)
```
Comment (jsoref) [inline pkg/controller/sparkapplication/controller.go:180]: Both flavors are used in this corpus. [Google Ngrams](https://books.google.com/ngrams/graph?content=enqueueing%2C+enqueuing&year_start=1800&year_end=2019&corpus=26&smoothing=3&direct_url=t1%3B%2Cenqueueing%3B%2Cc0%3B.t1%3B%2Cenqueuing%3B%2Cc0), enqueuing is winning.

File: spark-docker/conf/prometheus.yaml [hunk 8]
```diff
@@ -26,7 +26,7 @@ rules:
       app_namespace: "$1"
       app_id: "$2"
   # These come from the application driver if it's a structured streaming application
-  # Example: default/sstreaming.driver.spark.streaming.QueryName.inputRate-total
+  # Example: default/streaming.driver.spark.streaming.QueryName.inputRate-total
```
Comment (jsoref) [inline spark-docker/conf/prometheus.yaml:29]: I'm assuming this is right

File: sparkctl/cmd/client.go [hunk 9]
```diff
@@ -32,7 +32,7 @@ import (
 func buildConfig(kubeConfig string) (*rest.Config, error) {
 	// Check if kubeConfig exist
 	if _, err := os.Stat(kubeConfig); os.IsNotExist(err) {
-		// Try InclusterConfig for sparkctl running in a pod
+		// Try InClusterConfig for sparkctl running in a pod
```
Comment (jsoref) [inline sparkctl/cmd/client.go:35]: To match the next line

**Comment by liyinan926** [comment]
Thanks for fixing those!
